store/sqlite/guest_links.go

```diff
@@ -129,10 +133,11 @@ func guestLinkFromRow(row rowScanner) (picoshare.GuestLink, error) {
 	var maxFileBytes picoshare.GuestUploadMaxFileBytes
 	var maxFileUploads picoshare.GuestUploadCountLimit
 	var creationTimeRaw string
-	var expirationTimeRaw string
+	var urlExpirationTimeRaw string
+	var fileExpirationTimeRaw string
```

Comment:
>To implement your database suggestion I had to convert the NULL into 'NULL' as I kept getting: Scan error on column index 6, name "file_expiration_time": converting NULL to string is unsupported

The problem here is that it has to be a pointer to a string (`*string`)

And then after `row.Scan`, we'd want something like:

```go
var fileLifetime picoshare.FileLifetime
if (fileLifetimeRaw == nil) {
  fileLifetime = picoshare.FileLifetimeInfinite
} else {
  fileLifetime, err := parseFileLifetime(*fileLifetimeRaw)
  if err != nil {
    return picoshare.GuestLink{}, err
  }
}
```